eduVPN/Enroll2FAViewController.swift

```diff
@@ -46,11 +46,11 @@ class Enroll2FAViewController: NSViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // Change title color
-        let paragraphStyle = NSMutableParagraphStyle()
-        paragraphStyle.alignment = .center
-        let attributes = [NSAttributedString.Key.font: NSFont.systemFont(ofSize: 17), NSAttributedString.Key.foregroundColor : NSColor.white, NSAttributedString.Key.paragraphStyle : paragraphStyle]
-        doneButton.attributedTitle = NSAttributedString(string: doneButton.title, attributes: attributes)
+//        // Change title color
+//        let paragraphStyle = NSMutableParagraphStyle()
+//        paragraphStyle.alignment = .center
+//        let attributes = [NSAttributedString.Key.font: NSFont.systemFont(ofSize: 17), NSAttributedString.Key.foregroundColor : NSColor.white, NSAttributedString.Key.paragraphStyle : paragraphStyle]
+//        doneButton.attributedTitle = NSAttributedString(string: doneButton.title, attributes: attributes)
```

Comment:
COCWOCS!!!!! 😱 You!?!?! OMG!!!

Reply:
Oops! But cocwoc fix was here before you made the comment (but after I asked for review, my bad!) :-)